src/vivarium_gates_iv_iron/results_processing/process_results.py

```diff
@@ -101,46 +107,56 @@ def aggregate_over_seed(data):
     non_count_columns = []
     for non_count_template in results.NON_COUNT_TEMPLATES:
         non_count_columns += results.RESULT_COLUMNS(non_count_template)
-    count_columns = [c for c in data.columns if c not in non_count_columns + GROUPBY_COLUMNS]
+    count_columns = [
+        c for c in data.columns if c not in non_count_columns + GROUPBY_COLUMNS
+    ]
 
     # non_count_data = data[non_count_columns + GROUPBY_COLUMNS].groupby(GROUPBY_COLUMNS).mean()
     count_data = data[count_columns + GROUPBY_COLUMNS].groupby(GROUPBY_COLUMNS).sum()
-    return pd.concat([
-        count_data,
-        # non_count_data
-    ], axis=1).reset_index()
+    return pd.concat(
+        [
```

Comment:
not sure about all these extra lines. would it have let it all stay on one line if it had started on one line?

Reply:
Maybe. This is something to revisit when we apply Black to the research template.